Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.yml

```diff
@@ -125,6 +127,44 @@ configuration:
   - Unspecified
   required: false
   type: 15
+- additionalinfo: 'Fetches detection by either Rule ID (format: ru_{UUID}) or Version
+    ID (format: {ruleId}@v_{int64}_{int64}. Enter in comma separated format to add multiple.'
+  display: Detections to fetch by Rule ID or Version ID
+  hidden: false
+  name: fetch_detection_by_ids
+  required: false
+  type: 0
+- additionalinfo: Fetches all detections for the latest versions of all the live rules. At max 50
```

Comment:
Why would a limit of 50 be strongly enforced? I feel we should reconsider this choice

Reply:
@guykeller We have observed in our chronicle instance that there can be as many as `10000` or at times even more than that many numbers of detections associated with a `single rule`. And here we are providing the customer with an option to fetch all the detections associated with all the live rules in their chronicle instance. So to avoid overloading the demisto instance with the huge number of incidents coming in, we have restricted the `number of rules` to `50`.

Reply:
> Why would a limit of 50 be strongly enforced? I feel we should reconsider this choice

@guykeller We have observed in our chronicle instance that there can be as many as 10000 or at times even more than that many numbers of detections associated with a single rule. And here we are providing the customer with an option to fetch all the detections associated with all the live rules in their chronicle instance. So to avoid overloading the demisto instance with the huge number of incidents coming in, we have restricted the number of rules to 50.

Reply:
I suggest allowing the user to enter a number, use 50 as a default, and a strong restriction on 200.
But that's your call, depending on your knowledge with the product - just let me know what you decide

Reply:
> I suggest allowing the user to enter a number, use 50 as a default, and a strong restriction on 200.
> But that's your call, depending on your knowledge with the product - just let me know what you decide

@guykeller According to our product understanding and observations, we would like to go ahead with this strong restriction of 50, rather than adding a configuration parameter for the same. Please let us know if you would like to discuss this over a call.